Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1

```diff
@@ -0,0 +1,244 @@
+[CmdletBinding()]
+param(
+    [string] $SharePointCmdletModule = (Join-Path "C:\users\brian\Documents\GitHubVisualStudio\xSharePoint\Tests\xSharePoint" "..\Stubs\SharePoint\15.0.4693.1000\Microsoft.SharePoint.PowerShell.psm1" -Resolve)
```

Comment:
Where? Look at PR #2 - I see all those references removed and replaced with $PSScriptRoot